.github/scripts/trymerge.py

```diff
@@ -1261,9 +1261,9 @@ def find_matching_merge_rule(
             if num_matching_files > reject_reason_score:
                 reject_reason_score = num_matching_files
                 reject_reason = "\n".join(
-                    f"Not all files match rule `{rule_name}`."
-                    f"{num_matching_files} files matched, but there are still non-matching files:"
-                    f"{','.join(non_matching_files[:5])}{', ...' if len(non_matching_files) > 5 else ''}"
+                    f"Not all files match rule `{rule_name}`.",
+                    f"{num_matching_files} files matched, but there are still non-matching files:",
+                    f"{','.join(non_matching_files[:5])}{', ...' if len(non_matching_files) > 5 else ''}",
```

Comment:
If `ruff` does it, it's wrong here, as `join` does not take multiple arguments, but only one (at least on 3.9):
```
% python3.9 -c "print('-'.join('a', 'b', 'c'))"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: str.join() takes exactly one argument (3 given)
```

Reply:
No, it was me failing to solve a merge conflict with a commit from a PR that originated from a conversation from this PR: https://github.com/pytorch/pytorch/pull/101606#discussion_r1197146340. Note you made a review on an outdated version (see the following commit that fixes it).

Reply:
Oh, it was a conversation w you actually. But it wasn't ruff, it was my incorrect merge conflict resolution.